HHS/Head-Start-TTADP
# Change reports' status

## Description of change
This PR contains a script that deletes approved reports by changing their status to 'deleted'. It also includes changes to allow working of off the `HHS/main` branch directly without needing to use the `adhocteam ` fork. When a PR is opened for the `HHS/main` branch, the feature branch can be deployed to the Dev environment if configured in the `.config.yml`. The `HHS/main` branch will deploy to Staging as before.

## How to test
- yarn deleteApproved:local -i 555,777 (comma-separated list of ids)
- verify that the activity reports don't show in the table anymore
- to test the deployment to Dev, check that the script from this branch is deployed there

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-63
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-391


* Note:
The script is flexible to change status to a different status than deleted, however, the application currently doesn't support changing the status from approved to draft. If attempted, the legacy reports will not be able to be changed back to approved and non-legacy reports will not display the approvers' section correctly. I allowed this functionality with the idea that in the future we could use it if the UI for the non-legacy reports will be adjusted to show correctly.


## Checklists

### Every PR


- [n/a ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [ n/a] Logical Data Model updated
- [ n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
